js/app.js

```diff
@@ -230,8 +230,9 @@ app.controller('CheckoutController', function($scope, $interval, Order, $httpPar
                 $scope.clock = $scope.order.timestamp + totalTime - Math.floor(Date.now() / 1000);
                 //Mark order as expired if we ran out of time
                 if ($scope.clock < 0) {
-                    $scope.order.status = -3;
-                    return;
+					$scope.order.status = -3;
+					return;
+					}
```

Comment:
![67731774-ebd49680-fa01-11e9-8f82-6ed087871487](https://user-images.githubusercontent.com/39666372/68000913-9d760080-fc6a-11e9-822b-6078d1495f94.png)
This error is being caused by the extra `}` you have included. Please check your opening and closing tags. This should be removed as it is breaking the checkout on my end.

If you visit https://github.com/blockonomics/woocommerce-plugin/pull/160/files, you will notice all of your new code is shifted further right than the existing code. Please correct this.